### PR TITLE
Addition of the honorary delta 2024

### PR DESCRIPTION
**Pull request by olofbmelcherson**
Self explanitory

Added the recipients for 2024 + swedish motivations and updated the old recipient Christer Gesslers name as his last name has been changed

### CODE REVIEW

File: sektionen/hedersdelta/body.md [hunk 1]
```diff
@@ -1892,14 +1948,14 @@ Motivering borttappad.
...
 ## Motiveringar till 1999 års mottagare
 
-#### Christer Gessler <a name="christergessler"></a>
+#### Christer Eliasi-Swahn (Gessler) <a name="christergessler"></a>
 
 <em>Christer Gessler</em> har i tre år representerat Datasektionen
-ikårfullmäktige - två år som oberoende KF-ledamot och ett år som
+i kårfullmäktige - två år som oberoende KF-ledamot och ett år som
 Studienämndensordförande, SNO. I fullmäktige såväl som på sektionen har
 han med engagemangtagit ställning och argumenterat i relevanta frågor.
```
Comment (mathiasmagnusson) [inline sektionen/hedersdelta/body.md:1956]:
```suggestion
han med engagemang tagit ställning och argumenterat i relevanta frågor.
```
Nice att du fixade lite ihopskrivningar. Hittade dock en till :p

File: sektionen/hedersdelta/body.md [hunk 2]
```diff
@@ -787,6 +805,44 @@ går det även att maila D-rektoratet.
...
+
+Utöver sitt ordförandeskap under pandemin tog Niklas initiativ till att återuppliva en inaktiv idrottsgrupp genom att organisera gruppen och hålla rutinmässiga event samt pusha dessa. Detta ledde till en naturlig interaktion mellan programstudenter och internationella studenter samt ökat engagemang för idrott och hälsa inom sektionen.
+
+För sina insatser i att hålla sektionen vid liv under en tuff period och det bestående arbetet med att göra sektionen mer aktiv, tilldelas Niklas Vatn Hedersdeltat. 
+
+#### Amanda Berg <a name="amaberg"></a>
+
+Amanda Berg har under sin tid här på sektionen visat ett enastående engagemang för att tillse att jämlikhet, mångfald och likabehandling ska tas seriöst. I sin roll som Jämlikhetsnämndens Ordförande har Amanda uppnått detta genom att införa en JML policy för sektionen samt att kontinuerligt utveckla denna under sina efterföljande funktionärsposter. Detta har i sin helhet lett till en mer inkluderande och trivsam miljö för alla våra medlemmar. Amanda var även ansvarig för ett projekt som sträckte sig över sektionsgränserna vars syfte var att främja kvinnor och icke-binära på hela KTH och inte enbart inom sektionen.
```
Comment (mathiasmagnusson) [inline sektionen/hedersdelta/body.md:840]:
```suggestion
Amanda Berg har under sin tid här på sektionen visat ett enastående engagemang för att tillse att jämlikhet, mångfald och likabehandling ska tas seriöst. I sin roll som Jämlikhetsnämndens Ordförande har Amanda uppnått detta genom att införa en JML-policy för sektionen samt att kontinuerligt utveckla denna under sina efterföljande funktionärsposter. Detta har i sin helhet lett till en mer inkluderande och trivsam miljö för alla våra medlemmar. Amanda var även ansvarig för ett projekt som sträckte sig över sektionsgränserna vars syfte var att främja kvinnor och icke-binära på hela KTH och inte enbart inom sektionen. 
```

File: sektionen/hedersdelta/body.md [hunk 3]
```diff
@@ -787,6 +805,44 @@ går det även att maila D-rektoratet.
...
+
+För sitt ovärderliga arbete inom förvaltningen av sektionen samt sitt driv för att göra rätt sak även om det innebär uppoffringar för sig själv, tilldelas David Puustinen Hedersdeltat.
+
+#### Ebba Bråtman <a name="ebratman"></a>
+
+Ebba Bråtman har bidragit till sektionen på många olika sätt men framförallt inom mottagningen och JML arbete. Ebba har även varit aktiv inom flertalet nämnder och projekt bland annat Ada, D-Dagen och projektet Magenta. Under sin tid i Titel som Storquisine löste Ebba problem och samarbetade på ett tillmötesgående sätt vilket hon även bar med sig i rollen som Storasyster året därpå. Som Storasyster underlättade Ebba för personalen samt Titel genom att vidareutveckla det tidigare arbetet, i syfte att förbättra personalens välmående. Detta gjorde Ebba genom att strukturera upp schemaläggningen av personalen så att de kunde göra sitt bästa vid mottagandet av nØllan.
```
Comment (mathiasmagnusson) [inline sektionen/hedersdelta/body.md:824]:
```suggestion
Ebba Bråtman har bidragit till sektionen på många olika sätt men framförallt inom mottagningen och JML-arbete. Ebba har även varit aktiv inom flertalet nämnder och projekt bland annat Ada, D-Dagen och projektet Magenta. Under sin tid i Titel som Storquisine löste Ebba problem och samarbetade på ett tillmötesgående sätt vilket hon även bar med sig i rollen som Storasyster året därpå. Som Storasyster underlättade Ebba för personalen samt Titel genom att vidareutveckla det tidigare arbetet, i syfte att förbättra personalens välmående. Detta gjorde Ebba genom att strukturera upp schemaläggningen av personalen så att de kunde göra sitt bästa vid mottagandet av nØllan. 
```

**Comment by mathiasmagnusson** [comment]
Schmunguss @olofbmelcherson

**Review by Herkarl — APPROVED** [review]

**Comment by Herkarl** [comment]
Fuck, missade en kommentar

**Comment by foodelevator** [comment]
Men det var på de gamla sakerna, så inga nya stavfel lades till :p

**Pull request merged**